Source/Helpers/StringParser.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Globalization;
+
+namespace Svg.Helpers
+{
+    internal static class StringParser
+    {
+        private static readonly CultureInfo Format = CultureInfo.InvariantCulture;
+
+        public static float ToFloat(ref ReadOnlySpan<char> value)
+        {
+#if NETSTANDARD2_1 || NETCORE || NETCOREAPP2_1 || NETCOREAPP3_1 || NET5_0
```

Comment:
Considering the future, I think that the following will improve maintainability.
```cs
#if Net4
    return float.Parse(value.ToString(), NumberStyles.Float, Format);
#else
    return float.Parse(value, NumberStyles.Float, Format);
#endif
```

Reply:
NETSTANDARD2_0 does not support that API

Reply:
@wieslawsoltes 
Add NETSTANDARD2_0, too.
The newly added API will not be removed in the future.
However, as it is, it has to be added to `#if` every time the version of Framework goes up.

Reply:
It requires netstandard2.1

Reply:
```cs
#if Net4 || NETSTANDARD2_0
    return float.Parse(value.ToString(), NumberStyles.Float, Format);
#else
    return float.Parse(value, NumberStyles.Float, Format);
#endif
```

Reply:
Now yes.